### ISSUE TITLE
Toast is not accessibile on top of material-ui's modal when accessibility is on

### ISSUE THREAD

**Issue by CHANDRU-WMH** [fkhadra/react-toastify#948]
**Do you want to request a *feature* or report a *bug*?** BUG

When calling a toast on top modal toast is displayed but it's not announced as an alert when accessibility voice-over is turned on. Can't even click the toast when accessibility mode is turned on.
When we kept the toast container inside the modal, it was accessible and working as expected but we are calling toast in both react and non-react functions so it's not useful.
When using two containers one in modal and another in app.js at that time two toast is coming. Controlling by container id is also not useful since we are also calling in non-react functions. IS there any other approach to overcome this issue?

**Comment by fkhadra** [comment]
Hey @CHANDRU-WMH, a bit late but I've looked into the issue. This is happening because MUI will add `aria-hidden=true` to all body children. One solution to overcome the limitation is to use react-portal and to remove aria-hidden from the ToastContainer.  The other solution is the one you mentioned, adding the ToastContainer inside the dialog so it's not affected by the aria-hidden.
In a nutshell

```tsx

// 1. when triggering the notification
document.getElementById("containerId").removeAttribute("aria-hidden");
toast('msg')

// 2. when rendering the container
import { createPortal } from "react-dom";

<App>
{/* some content */}
{createPortal(<ToastContainer containerId="containerId" />, document.body)}
</App>
```` 

You can find an example here https://codesandbox.io/s/react-toastify-mui-accessibility-xw74vr
